C/ds/Primefactorization_Sieve.c

```diff
@@ -0,0 +1,66 @@
+# include <stdio.h> 
+# include <math.h> 
+
+void primeFactors(int number)                 // Program to print all prime factors of a number using prime sieve
+{ 
+	int primes[number+1];
+	int i,j;                                            
+                                                    
+    for(i = 2; i<=number; i++)              
+        primes[i] = i;
+
+    i = 2;
+    
+    while ((i*i) <= number)
+    {
+        if (primes[i] != 0)
+        {
+            for(j=2; j<number; j++)
+            {
+                if (primes[i]*j > number)
+                    break;
+                else
+                                                // marking the elements as 0
+                    primes[primes[i]*j]=0;
+            }
+        }
+        i++;
+    }
+
+   	while (number%2 == 0)                       // check if a number is even
+	{ 
+		printf("%d ", 2); 
+		number = number/2; 
+	} 
+
+
+	for (int i = 3; i <= sqrt(number); i = i+2) //the number must be odd 
+	{ 
+		                                        // While i divides n, print i and divide n 
```

Comment:
Use Multi-Line comments instead of Single-Line

Reply:
resolved the issue. Kindly review it.